lib/auth/token_store.dart

```diff
@@ -9,12 +9,14 @@ abstract class TokenStore {
 
   DateTime? get expiry => _token?._expiry;
 
+  String? get kind => _token?._kind;
```

Comment:
I've named it `kind` because this method: https://cloud.google.com/identity-platform/docs/reference/rest/v1/accounts/signInWithCustomToken returns `kind`, so I wanted to be 1-to-1 with the API.
Shall I still rename it?